src/aioswitcher/device/tools.py

```diff
@@ -158,3 +160,23 @@ def convert_str_to_devicetype(device_type: str) -> DeviceType:
     elif device_type == DeviceType.RUNNER_MINI.value:
         return DeviceType.RUNNER_MINI
     return DeviceType.MINI
+
+
+def get_token(username: str, password: str) -> DeviceToken:
+    """Make API call to get a Token by username and password."""
+    token = DeviceToken("")
+    url = "https://switcher.co.il/GetKey/GetTok/t.php"
+    data = {"username": username, "password": password}
+
+    response = requests.post(url, data=data)
+
+    if response.status_code == 200:
+        print("Request successful")
+        try:
+            token = DeviceToken(response.json())
+
+        except ValueError:
+            print("Response content is not valid JSON")
+    else:
+        print("Request failed with status code:", response.status_code)
+    return token
```

Comment:
Please do not use `print` in API calls. You already have prints in the script intended for the user. If we plan to use this API for automating the token these prints should be replaced by debug logger.

Reply:
Ok sure will do

Reply:
Done. replace print with logger